### DISCUSSION TITLE
Pending Transactions

### DISCUSSION THREAD

**Discussion by Joeypowertools** [blockscout/blockscout#9758] [General]
How long do pending transactions typically take?

0xa7286b7fa32a6bd14a24004c151b706f275c44115961dd03071dc23ca048fc0f

**Comment by Markfluit** [comment]
There’s a way you can find a solution to this as long as the transaction is still existing in your wallet.
Send am email to cryptorecoverys@outlook.com
For more assistance

**Comment by Joeypowertools** [comment]
![image](https://github.com/blockscout/blockscout/assets/119900780/432c81f8-27db-43c4-9c7f-c83c87672e78)

**Comment by Joeypowertools** [comment]
![image](https://github.com/blockscout/blockscout/assets/119900780/8cfd6c3a-8f17-4ea0-98db-4c402f18cd79)

**Reply by Markfluit** [reply]
I had similar issues with my transaction pending for some days….i can say you can quickly reach out to cryptorecoverys@outlook.com they helped me out.
